frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/basicTicketImage/basicTicketImage.styles.ts

```diff
@@ -15,34 +15,28 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { Typography } from '@mui/material';
+import { FormLabel } from '@mui/material';
 import styled, { css } from 'styled-components';
 
-export const Container = styled.div`
+export const Container = styled.div<{ error?: string }>`
 	padding: 13px;
 	border: solid 1px ${({ theme }) => theme.palette.secondary.lightest};
 	background-color: ${({ theme }) => theme.palette.primary.contrast};
 	border-radius: 5px;
 	display: flex;
 	flex-direction: row;
 	justify-content: space-between;
+
+	${({ error, theme }) => error && css`
+		border: solid 1px ${theme.palette.error.main};
+		background-color: ${theme.palette.error.lightest};
+	`}
 `;
 
-export const PropertyName = styled(Typography).attrs({
-	variant: 'h5',
-})<{ required?: boolean }>`
+export const PropertyName = styled(FormLabel)`
```

Comment:
Not ken on the name of the component PropertyName. At these level these are not exactly properties, but just specific types of inputs.